### ISSUE TITLE
ESSV3 Spawn Location

### ISSUE THREAD

**Issue by TrinityBob** [ebayShopper/ESSV3#18]
I have just now installed the latest version of this mod, and as with the previous version, all players spawn far outside the map. I use the latest version of Infistar (1448). I also need more information on where to add the BE exceptions.

**Comment by ebayShopper** [comment]
Which map are you running?

Please post any errors from the client and server RPTs

**Comment by TrinityBob** [comment]
Im using Chernarus on Epoch 1.0.6.2

I have only found two instances it the RPT files (server and client side). With me as admin i do not have this spawn issue but every other player on the server experiences this.

Cannot Sync Character <playername> near respawn_west [-18700,25800,0.00149536]. This is normal when relogging or changing clothes."
On a map that location is very far north West on the map. Far outside the bounds of playable region.
and also:

15:05:49 "INFO: OnPlayerDisconnect exiting. Player is not in playableUnits. ["<PlayerUID>","<Player Name>"]"

Thanks for having a look at this. It is very frustrating although it is a really awesome mod.

I also found some reference to this location above in the variables.sqf in the dayz_code.pbo 
respawn_west_original = getMarkerPos "respawn_west"; //Prevent problems caused by cheaters moving respawn_west marker with setMarkerPos or deleteMarker

switch (toLower worldName) do {
	case "napf";
	case "ruegen";
	case "sauerland" : {dayz_minpos = -1000; dayz_maxpos = 26000;};
	case "tavi" : {dayz_minpos = -26000; dayz_maxpos = 26000;};
	case "chernarus" : {dayz_minpos = -1; dayz_maxpos = 16000;};
	case default {dayz_minpos = -20000; dayz_maxpos = 20000;};

P.S. Im using the latest version AntiHack by InfiSTAR (1448v)
